src/options/flags.rs

```diff
@@ -11,7 +11,7 @@ pub static GRID:        Arg = Arg { short: Some(b'G'), long: "grid",        take
 pub static ACROSS:      Arg = Arg { short: Some(b'x'), long: "across",      takes_value: TakesValue::Forbidden };
 pub static RECURSE:     Arg = Arg { short: Some(b'R'), long: "recurse",     takes_value: TakesValue::Forbidden };
 pub static TREE:        Arg = Arg { short: Some(b'T'), long: "tree",        takes_value: TakesValue::Forbidden };
-pub static CLASSIFY:    Arg = Arg { short: Some(b'F'), long: "classify",    takes_value: TakesValue::Forbidden };
+pub static CLASSIFY:    Arg = Arg { short: Some(b'F'), long: "classify",    takes_value: TakesValue::Optional(Some(WHEN), "always") };
```

Comment:
having this doesnt make sens as the goal of a WHEN argument is to be able to without modifying your alias nor caring of what you add behing to pipe the output. Thats why auto is the base case. Here the classify will stay on if the output is piped. I know this create a breaking change but this is how we use, create, WHEN arguments

Reply:
I can definitely change the default to `auto`. As for my alias, I can hardcode `--classify=auto`. From a consistency perspective I prefer `auto` to have the same behavior as other flags. From backwards compatibility it will meen `--classify` will not longer add characters when piped, and people relying on that will need to switch to `--classify=always`. I think it is unlikely to have people relying on that, but I wanted to be conservative on the first version on the patch and get your input.